Firebase/Auth/Source/FIRAuthErrorUtils.m

```diff
@@ -345,6 +345,18 @@
 static NSString *const kFIRAuthErrorMessageCaptchaCheckFailed = @"The reCAPTCHA response token "
     "provided is either invalid, expired or already";
 
+/** @var kFIRAuthErrorMessageWebContextAlreadyPresented
+    @brief Message for @c FIRAuthErrorCodeWebContextAlreadyPresented error code.
+ */
+static NSString *const kFIRAuthErrorMessageWebContextAlreadyPresented = @"User interaction is "
+    "still ongoing, another view cannot be presented.";
+
+/** @var kFIRAuthErrorMessageWebContextCancelled
+    @brief Message for @c FIRAuthErrorCodeWebContextCancelled error code.
+ */
+static NSString *const kFIRAuthErrorMessageWebContextCancelled = @"The user cancelled the sign-in "
```

Comment:
It is not necessarily sign-in flow. How about "The interaction was cancelled by user."?

Reply:
Done.